frontend/src/metabase/visualizations/components/ChoroplethMap.jsx

```diff
@@ -126,8 +126,8 @@ export default class ChoroplethMap extends Component {
     };
   }
 
-  componentWillMount() {
-    this.componentWillReceiveProps(this.props);
+  UNSAFE_componentWillMount() {
+    this.UNSAFE_componentWillReceiveProps(this.props);
```

Comment:
we should really make this sort of thing a lint error one day imo, not a fan of reusing lifecycle methods like this

Reply:
Oh, absolutely. I didn't even notice this one.
Good catch.